po/common/common.pot

```diff
@@ -3688,3 +3688,63 @@ msgstr "Cancel"
 msgctxt "collapsed_changes"
 msgid "Collapsed changes"
 msgstr "Collapsed changes"
+
+msgctxt "data_quality_p"
+msgid "data quality"
+msgstr "data quality"
+
+msgctxt "data_quality_s"
+msgid "data quality"
+msgstr "data quality"
+
+msgctxt "data_quality"
+msgid "data quality"
+msgstr "data quality"
+
+msgctxt "data_quality_bugs_p"
+msgid "data quality bugs"
+msgstr "data quality bugs"
+
+msgctxt "data_quality_bugs_s"
+msgid "data quality bug"
+msgstr "data quality bug"
+
+msgctxt "data_quality_bugs"
+msgid "data quality bugs"
+msgstr "data quality bugs"
+
+msgctxt "data_quality_info_p"
+msgid "data quality info"
+msgstr "data quality info"
+
+msgctxt "data_quality_info_s"
+msgid "data quality info"
+msgstr "data quality info"
+
+msgctxt "data_quality_info"
+msgid "data quality info"
+msgstr "data quality info"
+
+msgctxt "data_quality_warnings_p"
+msgid "data quality warnings"
+msgstr "data quality warnings"
+
+msgctxt "data_quality_warnings_s"
+msgid "data quality warning"
+msgstr "data quality warning"
+
+msgctxt "data_quality_warnings"
+msgid "data quality warnings"
+msgstr "data quality warnings"
+
+msgctxt "data_quality_errors_p"
+msgid "data quality errors"
+msgstr "data quality errors"
+
+msgctxt "data_quality_errors_s"
+msgid "data quality error"
+msgstr "data quality error"
+
+msgctxt "data_quality_errors"
+msgid "data quality errors"
+msgstr "data quality erros"
```

Comment:
Small typo: s/erros/errors/

Reply:
Fixed, thank you!
BTW did you know that you can make suggestions like that in github? Like in a google doc suggestion, they can be accepted in one click.
![image](https://user-images.githubusercontent.com/8158668/66748761-47424880-ee88-11e9-92be-e18543a955d1.png)

Reply:
Oh my!... thanks @stephanegigandet !